Makefile

```diff
@@ -1026,6 +1026,12 @@ compare: out/mkcmp out/minikube
 generate-licenses:
 	./hack/generate_licenses.sh
 
+.PHONY: gomodtidy
+gomodtidy: ## run go mod tidy everywhere needed
+	go mod tidy
+	(cd hack && go mod tidy)
```

Comment:
We don't need to a sub shel since every command is using a sub shell.

I tried this:

```
check:
	pwd
	cd hack
	pwd

% make check
pwd
/Users/nsoffer/src/minikube
cd hack
pwd
/Users/nsoffer/src/minikube
``

Reply:
I will remove this one :) thanks